tests/ballerina-test-integration/src/test/resources/httpService/httpHeaderTest.bal

```diff
@@ -37,18 +37,11 @@ service<http:Service> headerService bind headerServiceEP {
 
         match reply {
             http:Response clientResponse => {
-                var result = clientResponse.getHeaders("person");
-                match result {
-                    string[] headers => {
-                        json payload = {header1:headers[0] , header2:headers[1]};
-                        http:Response res = {};
-                        res.setJsonPayload(payload);
-                        _ = conn -> respond(res);
-                    }
-                    any|null => {
-                        return;
-                    }
-                }
+                string[] headers = clientResponse.getHeaders("person");
```

Comment:
you may need to first check, and then get. clientResponse.hasHeaders("person"). Polite programming :)

Reply:
will do the change :)